### PR TITLE
Update to SPIRE 1.8.4

### PR DESCRIPTION
**Pull request by faisal-memon**
fixes: https://github.com/spiffe/helm-charts-hardened/issues/71

### CODE REVIEW

File: charts/spire/README.md.gotmpl [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 | Dependency | Supported Versions |
 |:-----------|:-------------------|
-| SPIRE      | `1.5.3+`, `1.6.3+` |
+| SPIRE      | `1.8.4`            |
```
Comment (kfox1111) [inline charts/spire/README.md.gotmpl:22]: I think this whole file may not be used anymore? Should we get rid of it?

**Review by kfox1111 — APPROVED** [review]
Looks good to me. One issue identified, but we could also do that in a followup PR.

File: charts/spire/README.md.gotmpl (hunk 1 above)
Comment (marcofranssen) [reply charts/spire/README.md.gotmpl:22]: correct. This got replaced when we switched doc tool

**Review by marcofranssen — APPROVED** [review]
LGTM 🚀

File: charts/spire/README.md.gotmpl (hunk 1 above)
Comment (kfox1111) [reply charts/spire/README.md.gotmpl:22]: Filed https://github.com/spiffe/helm-charts-hardened/issues/87

Comment (marcofranssen) [reply charts/spire/README.md.gotmpl:22]: https://github.com/spiffe/helm-charts-hardened/pull/86

**Pull request merged**